tests/bug-7126/input.rules

```diff
@@ -0,0 +1 @@
+alert ip any any -> any any (msg:"Sample rule"; flow: established, to_server; file_data; content:"no-match"; sid:1;)
```

Comment:
Do we need a rule for this test ?

Reply:
@jlucovsky what was your intention here ?

Reply:
The goal was to have the BDAT section of the pcap parsed. This rule caused that to happen and hence, demonstrate the issue and the fix.

Reply:
> The goal was to have the BDAT section of the pcap parsed.

Why is the BDAT not parsed if we do not have any rule ? 🤔